docs/source/tutorials/token_classification.ipynb

```diff
@@ -173,51 +178,56 @@
-    "                continue\n",
-    "            splits = line.split(sep) \n",
-    "            word = splits[0]\n",
-    "            if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
-    "                word = word[0] + word[1:].lower()\n",
-    "            sentence.append(word)\n",
-    "            label.append(entity_map[splits[-1][:-1]])\n",
+    "    data, sentence, label = [], [], []\n",
+    "    for line in lines:\n",
+    "        if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "            if len(sentence) > 0:\n",
+    "                data.append((sentence, label))\n",
+    "                sentence, label = [], []\n",
+    "            continue\n",
+    "        splits = line.split(sep) \n",
+    "        word = splits[0]\n",
+    "        if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
+    "            word = word[0] + word[1:].lower()\n",
+    "        sentence.append(word)\n",
+    "        label.append(entity_map[splits[-1][:-1]])\n",
     "\n",
-    "        if len(sentence) > 0:\n",
-    "            data.append((sentence, label))\n",
+    "    if len(sentence) > 0:\n",
+    "        data.append((sentence, label))\n",
     "\n",
-    "        given_words = [d[0] for d in data] \n",
-    "        given_labels = [d[1] for d in data] \n",
+    "    given_words = [d[0] for d in data] \n",
+    "    given_labels = [d[1] for d in data] \n",
     "\n",
-    "        return given_words, given_labels \n",
+    "    return given_words, given_labels \n",
     "\n",
-    "    given_words, given_labels = [], [] \n",
+    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
+    "given_words, given_labels = [], [] \n",
     "\n",
-    "    for filepath in filepaths: \n",
-    "        words, label = readfile(filepath) \n",
-    "        given_words.extend(words) \n",
-    "        given_labels.extend(label)\n",
+    "for filepath in filepaths: \n",
+    "    words, label = readfile(filepath) \n",
+    "    given_words.extend(words) \n",
+    "    given_labels.extend(label)\n",
     "\n",
-    "    sentences = list(map(get_sentence, given_words)) \n",
+    "sentences = list(map(get_sentence, given_words)) \n",
     "\n",
-    "    sentences, mask = filter_sentence(sentences) \n",
-    "    given_words = [words for m, words in zip(mask, given_words) if m] \n",
-    "    given_labels = [labels for m, labels in zip(mask, given_labels) if m] \n",
+    "sentences, mask = filter_sentence(sentences) \n",
+    "given_words = [words for m, words in zip(mask, given_words) if m] \n",
+    "given_labels = [labels for m, labels in zip(mask, given_labels) if m] \n",
     "\n",
-    "    maps = [0, 1, 1, 2, 2, 3, 3, 4, 4] \n",
-    "    given_labels = [mapping(labels, maps) for labels in given_labels] \n",
+    "maps = [0, 1, 1, 2, 2, 3, 3, 4, 4] \n",
+    "labels = [mapping(labels, maps) for labels in given_labels]\n",
```

Comment:
Here, I updated the variable name in the markdown code snippet (`given_labels -> labels`).